zipkin/src/main/java/com/linecorp/armeria/internal/tracing/SpanContextUtil.java

```diff
@@ -43,5 +44,13 @@ public static void setupContext(FastThreadLocal<SpanInScope> threadLocalSpan, Re
         });
     }
 
+    /**
+     * Closes a {@link Span} which related to a {@link RequestLog}.
```

Comment:
Adds logging tags to the provided {@link Span} and closes it. The span cannot be used further after this method has been called.